### ISSUE TITLE
Do you have any built in minifiers for removing whitespace from html bundles?

### ISSUE THREAD

**Issue by lukeapage** [BladeRunnerJS/brjs#1200]
Do you have anything built in to remove whitespace from HTML ?

For the purposes of minification and also avoiding inline-block whitespace issues (as an alternative to using `font-size: 0` or editing the templates to not have whitespace.

**Comment by andyberry88** [comment]
Hi @lukeapage,

At the moment the only minifier support we have is for JavaScript. We have spoken about minification of HTML and other resources but there's no ETA on this at the moment.

I've previously created https://github.com/BladeRunnerJS/brjs/issues/1012 which suggests a content transform plugin instead of the current minifier plugin which could be used to minify HTML and possibly run image optimizations during the bundling process.
A mechanism similar to this is needed to allow JNDI token like support for static file export so it's possibly something that may be looked at within the coming months although I can't make any promises.

**Comment by lukeapage** [comment]
fair enough, its probably outside our timescales, though we would use the minification for a small effect on data transferred if implemented.
